app/models/chargeback_rate.rb

```diff
@@ -70,4 +72,9 @@ def self.seed
       end
     end
   end
+
```

Comment:
@GregP that's fine. We don't need to write code for the sake of consistency.  If this statement is true:

> any place we try to delete the model, we'll have to reproduce this logic

Then we should use the `before_destroy` callback as @imtayadeway suggested.